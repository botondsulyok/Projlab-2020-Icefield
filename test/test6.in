field b1 3 false 0
stats b1
entity ib1 icebear b1
stats b1
entity ib2 icebear b1
stats b1
entity ib3 icebear b1
stats b1
entity ib4 icebear b1
stats b1
entity ib5 icebear b1
stats b1