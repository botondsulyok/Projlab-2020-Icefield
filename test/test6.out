id: b1
type: unstable
building: plains0
snow: 0
pickable: null
standers: null
pullers: null
masters: null
neighbours: null
capacity: 3
broken: false

id: b1
type: unstable
building: plains0
snow: 0
pickable: null
standers: ib1
pullers: null
masters: null
neighbours: null
capacity: 3
broken: false

id: b1
type: unstable
building: plains0
snow: 0
pickable: null
standers: ib1, ib2
pullers: null
masters: null
neighbours: null
capacity: 3
broken: false

id: b1
type: unstable
building: plains1
snow: 0
pickable: null
standers: ib1, ib2, ib3
pullers: null
masters: null
neighbours: null
capacity: 3
broken: true

id: b1
type: unstable
building: plains1
snow: 0
pickable: null
standers: ib1, ib2, ib3, ib4
pullers: null
masters: null
neighbours: null
capacity: 3
broken: true

id: b1
type: unstable
building: plains1
snow: 0
pickable: null
standers: ib1, ib2, ib3, ib4, ib5
pullers: null
masters: null
neighbours: null
capacity: 3
broken: true

